relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics_all_modules.snap

```diff
@@ -951,4 +951,30 @@ expression: metrics
             "span.op": "resource.script",
         },
     },
+    Bucket {
+        timestamp: UnixTimestamp(1702474613),
+        width: 0,
+        name: "d:spans/exclusive_time_light@millisecond",
+        value: Distribution(
+            [
+                32.000065,
+            ],
+        ),
+        tags: {
+            "span.description": "my-component-name",
+            "span.group": "e674f9eca1d88a4d",
+            "span.op": "ui.interaction.click",
```

Comment:
@0Calories I forgot that we now have stricter metrics extraction rules, so we need to specify exactly what metrics and tags we want for these interaction spans. For now, I started with just these three, but let me know if other tags are needed in the product.

Reply:
thanks @jjbayer , these are fine for now but in the future when we need to support more usecases we may need more metrics. Will reach out to you later when that time comes